LICENSE_header.txt

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
Do we need this file?

Reply:
Yes, in ADAM we have a build rule that confirms every source file has a proper license header.

Reply:
So every file should include the license header? That it's a bit redundant, but fine to me...

Reply:
Yes, it's pretty standard to do this.

Reply:
I know that it is kind of standard, but I always doubt about adding it or not because of redundancy and easier out-of-date years when updating. If there is a rule for checking that, I am not oposed to it.

Reply:
Why not just specify in the root that all code is under the given license?  I've never understood the need for individual license headers in every file, it's just a source of hassle and inconsistency.    A build rule helps but it seems better to just say "all files in this repo are under this license".

Reply:
I've found lots of code "imported" into other projects without properly adhering to the license.  If there is a license header on every file, it makes it explicit to the "importer" that they are breaking the license, having to remove the text of the license header.

Reply:
I lean towards having a header for exactly this reason.

Reply:
Hmn.  I see the point there, but it still seems like more hassle for us than there is benefit.